pkgs/servers/bloat/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, buildGoModule
+, fetchgit
+, unstableGitUpdater
+}:
+
+buildGoModule {
+  pname = "bloat";
+  version = "unstable-2022-02-11";
+
+  src = fetchgit {
+    url = "git://git.freesoftwareextremist.com/bloat";
+    rev = "c390a0c32720f6afe852bc7a3a3f64c3afe9e401";
+    sha256 = "sha256-SeIpm2zDnjllnsr+It5rIem+WNKRoJH1WqJuEL2hkRo=";
+  };
+
+  vendorSha256 = null;
```

Comment:
Why no `vendorSha256`?

Reply:
https://nixos.org/manual/nixpkgs/stable/index.html#ssec-language-go

> vendorSha256 can also take null as an input. When null is used as a value, rather than fetching the dependencies and vendoring them, we use the vendoring included within the source repo